<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QMainWindow" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1499</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1551</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 7, 0);</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font:  18pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>Welcome to The Gaming Store!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QPushButton" name="Next_store">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>650</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>401</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/image/welcome.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1499</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
